### ISSUE TITLE
Stripping out inline source maps

### ISSUE THREAD

**Issue by andrewjmead** [sindresorhus/gulp-autoprefixer#1]
I though it was an issue with `gulp-sass`, but it turns out that `gulp-autoprefixer` was the culprit.

## Problematic Task
```
gulp.task('css:dev', function () {
    return gulp.src('public/css/core.scss')
        .pipe(sass({sourceComments: 'map', errLogToConsole: true}))
        .pipe(autoprefix())
        .pipe(gulp.dest('public/css/'));
});
```

## Issue
After the scss is compiled into css, it has inline source maps that look like the following
```
/*# sourceMappingURL=data:application/json;base64,[base64 data]*/
```
Once that file is run through autoprefixer, the source maps don't exist. I wasn't able to find any documenation

**Comment by polarathene** [comment]
Pretty sure the issue is related to libsass outputting an incorrect sourcemap, autoprefixer runs fine when provided a proper sourcemap. Check that without autoprefixer that your sourcemap in devtools maps to the correct line in .scss, in my case it was either null or the last line of the file. It'll be fixed with libsass 3.2, more info [here](https://github.com/sass/node-sass/issues/619)

**Comment by polarathene** [comment]
@andrewjmead @aaronjensen @antwan1986 @valdelama There was an update to `gulp-sass` today, it supports the v2 beta of `node-sass` and in turn `libsass 3.1-beta`. The fixed mappings and bugfixes should solve this issue.

I've provided code examples that worked for me for inline and external sourcemaps with embedded or external sources. You can see them [here](https://github.com/sindresorhus/gulp-autoprefixer/issues/10#issuecomment-69910254).
